docs/BUILD.md

```diff
@@ -0,0 +1,82 @@
+# Building the App for App Store Submission
+
+First, clone the repo (requires `git` - see [Dependencies](#dependencies)):
+
+```
+git clone https://github.com/WorldHealthOrganization/app.git
+cd app
+```
+
+## iOS
+
+These instructions assume that an app record has already been added to the App Store Connect account. If you haven't done so already, you can follow [these instructions](https://help.apple.com/app-store-connect/#/dev2cd126805).
+
+### Install Dependencies
+
+`cd` into the `client/flutter/ios` directory:
+
+```
+cd client/flutter/ios
+```
+
+Install CocoaPods dependencies (requires `CocoaPods` - see [Dependencies](#dependencies)):
+
+```
+pod install
```

Comment:
````
$ cd client/flutter/ios
$ pod install
Ignoring iStats-1.6.1 because its extensions are not built.  Try: gem pristine iStats --version 1.6.1

[!] Invalid `Podfile` file: Generated.xcconfig must exist. If you're running pod install manually, make sure flutter pub get is executed first.
````

Reply:
what is your version of ruby and CocoaPods? it looks like they'll have to install flutter for Android anyway, so this may not be a problem, but just checking

Reply:
$ ruby --version
ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]
$ pod --version
Ignoring iStats-1.6.1 because its extensions are not built.  Try: gem pristine iStats --version 1.6.1
1.9.1

Reply:
also, Pat prepared #289 to check the dependencies directly into the repo just in case there were any challenges with CocoaPods. opens up the question of storing the dependencies here, which I'm not sure how you feel about. but it would definitely make it easiest for them

Reply:
hmm... worked fine for me without this step with ruby `2.7.0` and CocoaPods `1.9.1`

Reply:
works with `flutter pub get` in client/flutter